roles/monitoring/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+---
+# tasks file for monitoring
+
+  - name: Install Logwatch
+    apt: 
+      pkg: logwatch 
+      state: installed
+
+  - name: Configure Logwatch
+    command: logwatch --detail Medium --mailto {{ Email }} --output mail --service All --range 'between -7 days and today'  
+     
+# Installation of chkrootkit
+
+  - name: Install chkrootkit
+    apt:
+      pkg: chkrootkit
+      state: present
+
+# Create a directory for chkrootkit
+  
+  - file: 
+      path: /etc/chkrootkit 
+      state: directory
+      mode: 0755 
+
+# Update chkrootkit configuration
+ 
+  - name: Update chkrootkit config
+    template:
+      src: chkrootkit.conf.j2 
+      dest: /etc/chkrootkit/chkrootkit.conf
+      owner: root
+      group: root
+      mode: 0644
+
+# Run Chkrootkit
+
+  - name: Run chkrootkit
+    shell: chkrootkit
+
+# Search for open ports with nmap
+
+  - apt:
+      pkg: nmap
+      state: latest
+
+  - name: Scan for open ports
+    command: nmap -v -sT '{{ IP }}'
```

Comment:
kai pali mporeis na kaneis register to output gia na to xrisimopoiiseis somehow....